lib/main.js

```diff
@@ -11,21 +11,29 @@ function Client (ikey, skey, host) {
   this.host = host
 }
 
-Client.prototype.apiCall = function (method, path, params, callback) {
+Client.prototype.apiCall = function (method, path, params, callback, signature = 2) {
   var date = new Date().toUTCString()
   var headers = {
     'Date': date,
     'Host': this.host,
     'User-Agent': `duo_api_nodejs/${_PACKAGE_VERSION}`
   }
-  headers['Authorization'] = duo_sig.sign(
-    this.ikey, this.skey, method, this.host, path, params, date)
-
-  var qs = querystring.stringify(params)
   var body = ''
+  var qs = querystring.stringify(params)
+  if (signature === 5) {
```

Comment:
In our other clients, the sig version value was stored as a property of the client, and the canonicalization and signing functions take it as a parameter.  While I am not 100% sold on that scheme, it might be best if this stayed consistent with that if possible.  would that be feasible here?

Reply:
Moved into the client to be set at initialization. For now, I left the default signature version at 2.